include/seqan3/range/container/concept.hpp

```diff
@@ -142,16 +146,30 @@ namespace seqan3
  */
 //!\cond
 template <typename type>
-concept bool container_concept = requires (type val, type val2)
+concept bool container_concept = requires (type val, type val2, type const cval, typename type::iterator it)
 {
     // member types
     typename type::value_type;
     typename type::reference;
     typename type::const_reference;
-    typename type::iterator; //TODO must satisfy forward_iterator_concept and convertible to const_interator
-    typename type::const_iterator; //TODO must satisfy forward_iterator_concept
+
+    typename type::iterator;
```

Comment:
delete. you do not need this line since in the line below already assume that `typename type::iterator` compiles.

Reply:
I agree, it's not necessary. But I think it's clearer this way. @h-2 Any feelings on this?